draft.install

```diff
@@ -42,6 +43,13 @@ function draft_update_8103(&$sandbox = NULL) {
   \Drupal::service('module_installer')->install(['admin_toolbar_links_access_filter']);
 }
 
+/**
+ * Enable Paragraphs module.
+ */
+function draft_update_8601(&$sandbox = NULL) {
+  \Drupal::service('module_installer')->uninstall(['paragraphs']);
```

Comment:
Probaby `\Drupal::service('module_installer')->install(['paragraphs']);`, but there is no need to write updates at this point inho

Reply:
Removed unnecessary update function